wiki/Featured_Artists/es.md

```diff
@@ -0,0 +1,11 @@
+# Artistas destacados
+
+**¡Los artistas destacados** son artistas musicales que han trabajado con osu! para hacer una selección de sus pistas disponibles para uso sin restricciones en la comunidad. ¡Algunos de los artistas destacados también han compuesto pistas específicamente para osu! y los solté como osu! exclusivas
```

Comment:
looks like missing punctuation at the end but I also don't know spanish /shrug

Reply:
Added the punctuation